app-vite/lib/config-tools.js

```diff
@@ -117,9 +117,7 @@ function createViteConfig (quasarConf, quasarRunMode) {
       polyfillModulePreload: build.polyfillModulePreload,
       emptyOutDir: false,
       minify: build.minify,
-      sourcemap: build.sourcemap === true
-        ? 'inline'
-        : build.sourcemap || false
+      sourcemap: build.sourcemap ?? false
```

Comment:
The logic here that build.sourcemap should pass straight through to Vite. In the event it is undefined, false is the default value.